### ISSUE TITLE
Allow setting `Union` in UserSubSchema

### ISSUE THREAD

**Issue by m-wynn** [okta/terraform-provider-okta#142]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

While the `union` field exists in the [UserSubSchema](https://github.com/terraform-providers/terraform-provider-okta/blob/aad0d06214d612713e7b6299cab38dcab88f82d1/sdk/user_schema_models.go#L45) struct, there is no way to set it.  As a result, the "Group Priority" is always reset to "Use Group Priority" (union="DISABLE"), instead of "Combine values across groups" (union="ENABLE"), even if it had otherwise been set to enabled before.

![Screenshot from 2020-06-29 10-55-01](https://user-images.githubusercontent.com/2483388/86039428-455ced00-b9f7-11ea-9245-755b5ff52891.png)

### New or Affected Resource(s)

* okta_app_user_schema

### Potential Terraform Configuration

```hcl
resource "okta_app_user_schema" "my_roles" {
  app_id      = okta_app_saml.my_app.id
  index       = "roles"
  title       = "roles"
  type        = "array"
  permissions = "READ_WRITE"
  master      = "PROFILE_MASTER"
  array_type  = "string"
  array_enum  = values(local.all_my_roles)

  dynamic "array_one_of" {
    for_each = local.all_my_roles
    content {
      const = array_one_of.value
      title = array_one_of.key
    }
  }

  union = true # <-- this
}

```

### References

* If it were documented, it would be https://developer.okta.com/docs/reference/api/schemas/

**Comment by m-wynn** [comment]
This is still an issue that we are encountering.

**Comment by bogdanprodan-okta** [comment]
Hi @m-wynn! I've created PR and `union` attr will be available in the upcoming v3.8.0.
